.changeset/wicked-dragons-brush.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-techdocs': minor
+---
+
+Use same initial filter `owned` for the `TechDocsIndexPage` as for the `CatalogPage`.
```

Comment:
Maybe a note in here about "if you'd prefer to keep the default filter to `all`, do [x, y, z]`" could be useful?  It'd be nice to at least highlight that it's (more easily?) configurable now.